x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/lib/absolute_timing_to_relative_timing.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * READ ME:
+ *
+ * ILM policies express data age thresholds as minimum age from an absolute point of reference.
+ * The absolute point of reference could be when data was created, but it could also be when
+ * rollover has occurred. This is useful for configuring a policy, but when trying to understand
+ * how long data will be in a specific phase, when thinking of data tiers, it is not as useful.
+ *
+ * This code converts the absolute timings to _relative_ timings of the form: 30 days in hot phase,
+ * 40 days in warm phase then forever in cold phase.
+ */
+
+import moment from 'moment';
+import { flow } from 'fp-ts/lib/function';
+import { i18n } from '@kbn/i18n';
+
+import { splitSizeAndUnits } from '../../../lib/policies';
+
+import { FormInternal } from '../types';
+
+type MinAgePhase = 'warm' | 'cold' | 'delete';
+
+type Phase = 'hot' | MinAgePhase;
+
+const i18nTexts = {
+  forever: i18n.translate('xpack.indexLifecycleMgmt.relativeTiming.Forever', {
+    defaultMessage: 'Forever',
```

Comment:
nit: I think we should use a lowercase 'forever' for both the label and the string itself